resources/nl-core/nl-core-MedicationAdministration2.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="nl-core-MedicationAdministration2" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicationAdministration2" />
+  <name value="NlcoreMedicationAdministration2" />
+  <title value="nl core MedicationAdministration2" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <purpose value="A derived profile from [zib-MedicationAdministration2](http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration2) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="Medication-Process-v9-2-0-0" />
+    <uri value="https://decor.nictiz.nl/art-decor/decor-datasets--mp-?id=&amp;amp;amp;effectiveDate=&amp;amp;amp;conceptId=&amp;amp;amp;conceptEffectiveDate=" />
+    <name value="Medication Process 9 v2.0.0" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="MedicationAdministration" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration2" />
+  <derivation value="constraint" />
+  <differential>

```

Comment:
Moeten op alle in de zib al bestaande elementen ook niet mp-dataelement920-mappings worden aangebracht?

Reply:
Erg lastig om bij gebrek hier aan te checken of alles wel volledig is.

Reply:
>Moeten op alle in de zib al bestaande elementen ook niet mp-dataelement920-mappings worden aangebracht?

Goed punt :( tot voorheen niet gedaan. Dat is me ook wel een werkje. Opzich kan vanuit de MP9 dataset ook gezien worden welke concepten op zib concepten gebaseerd/geerfd zijn.

Reply:
Zal der ook een issue van maken.

Reply:
https://github.com/Nictiz/Nictiz-R4-zib2020/issues/120